src/main/charts/confluence/templates/serviceaccount.yaml

```diff
@@ -9,6 +9,12 @@ metadata:
   annotations:
     {{- toYaml . | nindent 4 }}
   {{- end }}
+  {{- if .Values.confluence.s3AttachmentsStorage.eksIrsa.roleArn }}
+  {{- if not .Values.serviceAccount.annotations }}
```

Comment:
It's tricky to squeeze anything in `with` block, so as a workaround checking if service account annotations are defined to avoid `annotations` duplication.